apps/services/utils.py

```diff
@@ -0,0 +1,76 @@
+from apps.common.exceptions import BackendException
+from apps.users.services import student_list_service
+
+from collections import defaultdict
+from fastapi import status as http_status
+import xlrd
+
+
+def check_content_type(file):
```

Comment:
pls add annotations
rename to check_file_content_type

Reply:
ok